### DISCUSSION TITLE
[Feature] Remote server access

### DISCUSSION THREAD

**Discussion by dominotrix** [zed-industries/zed#6712] [Feature Requests]
Hello,

I currently work with a very lightweight editor that has implemented a remote server access (via FTP) feature. I have around 100+ websites that I can instantly connect and edit their files. A simple double click on a file opens it. I can make any changes I want and just by pressing Ctrl+S the file is automatically uploaded on the web server.

This is very helpful since I am not working locally, but directly on the server. Is it possible that ZED will have a similar feature?

**Comment by Moshyfawn** [comment]
Follow #5347

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
